### PR TITLE
Release v0.0.3

### PR DESCRIPTION
**Pull request by bartoszmajsak**

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
After rebasing on `master` I will push the tag.

File: docs/modules/ROOT/pages/release_notes/v0.0.3.adoc [hunk 1]
```diff
@@ -17,3 +17,61 @@ Example:
...
+
+ * fix(tests): controls tp version env var (https://github.com/Maistra/istio-workspace/pull/392[#392]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * fix(operator): remove args and command from telepresence clone (https://github.com/Maistra/istio-workspace/pull/393[#393]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * fix(docs): proper install snippet [skip-e2e] (https://github.com/Maistra/istio-workspace/pull/388[#388]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * fix(operator): give istio-workspace SA auth to create metrics service (https://github.com/Maistra/istio-workspace/pull/387[#387]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * fix: cleans up linter usage (https://github.com/Maistra/istio-workspace/pull/385[#385]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * fix(e2e): add to the members array instead of replacing it (https://github.com/Maistra/istio-workspace/pull/380[#380]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * fix(test): generate a suitable deployment resource for a pure k8s ins… (https://github.com/Maistra/istio-workspace/pull/376[#376]), by https://github.com/aslakknutsen[@aslakknutsen]
```
Comment (aslakknutsen) [inline docs/modules/ROOT/pages/release_notes/v0.0.3.adoc]: Move to Test section?

File: docs/modules/ROOT/pages/release_notes/v0.0.3.adoc [hunk 2]
```diff
@@ -17,3 +17,61 @@ Example:
...
+ * test(generator): use generator in int test scenario (https://github.com/Maistra/istio-workspace/pull/386[#386]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * feature(test): add full dns support to host names in test-generator (https://github.com/Maistra/istio-workspace/pull/375[#375]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * feature(operator): find target hostnames via Services instead of nami… (https://github.com/Maistra/istio-workspace/pull/378[#378]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * feat(github): introduces label sync action (https://github.com/Maistra/istio-workspace/pull/374[#374]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * chore: minor refactoring - context and logger (https://github.com/Maistra/istio-workspace/pull/379[#379]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * feat: logs version when starting the operator (https://github.com/Maistra/istio-workspace/pull/344[#344]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+
+=== Bug fixes
+
+ * fix(tests): controls tp version env var (https://github.com/Maistra/istio-workspace/pull/392[#392]), by https://github.com/bartoszmajsak[@bartoszmajsak]
```
Comment (aslakknutsen) [inline docs/modules/ROOT/pages/release_notes/v0.0.3.adoc]: Move to Test section?

File: docs/modules/ROOT/pages/release_notes/v0.0.3.adoc [hunk 3]
```diff
@@ -17,3 +17,61 @@ Example:
 `ratings` vs `ratings.namespace.svc.cluster.local`
 
 == {nbsp}
+
+=== New features
+
+ * fix(release): improvements in release shell script [skip-e2e] (https://github.com/Maistra/istio-workspace/pull/396[#396]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * feat(cmd): version cmd prints short semver info [skip-e2e] (https://github.com/Maistra/istio-workspace/pull/391[#391]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * test(generator): use generator in int test scenario (https://github.com/Maistra/istio-workspace/pull/386[#386]), by https://github.com/aslakknutsen[@aslakknutsen]
```
Comment (aslakknutsen) [inline docs/modules/ROOT/pages/release_notes/v0.0.3.adoc]: Move to Test section?

File: docs/modules/ROOT/pages/release_notes/v0.0.3.adoc [hunk 4]
```diff
@@ -17,3 +17,61 @@ Example:
...
+ * feature(test): add full dns support to host names in test-generator (https://github.com/Maistra/istio-workspace/pull/375[#375]), by https://github.com/aslakknutsen[@aslakknutsen]
```
Comment (aslakknutsen) [inline docs/modules/ROOT/pages/release_notes/v0.0.3.adoc]: Updated PR Subject to match Commit, regen?

File: docs/modules/ROOT/pages/release_notes/v0.0.3.adoc [hunk 5]
```diff
@@ -17,3 +17,61 @@ Example:
...
+ * feat: logs version when starting the operator (https://github.com/Maistra/istio-workspace/pull/344[#344]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+
+=== Bug fixes
+
+ * fix(tests): controls tp version env var (https://github.com/Maistra/istio-workspace/pull/392[#392]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * fix(operator): remove args and command from telepresence clone (https://github.com/Maistra/istio-workspace/pull/393[#393]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * fix(docs): proper install snippet [skip-e2e] (https://github.com/Maistra/istio-workspace/pull/388[#388]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * fix(operator): give istio-workspace SA auth to create metrics service (https://github.com/Maistra/istio-workspace/pull/387[#387]), by https://github.com/aslakknutsen[@aslakknutsen]
+ * fix: cleans up linter usage (https://github.com/Maistra/istio-workspace/pull/385[#385]), by https://github.com/bartoszmajsak[@bartoszmajsak]
+ * fix(e2e): add to the members array instead of replacing it (https://github.com/Maistra/istio-workspace/pull/380[#380]), by https://github.com/aslakknutsen[@aslakknutsen]
```
Comment (aslakknutsen) [inline docs/modules/ROOT/pages/release_notes/v0.0.3.adoc]: Move to Test section?

**Pull request merged**